tests/functional/aws-node-sdk/test/legacy/authV2QueryTests.js

```diff
@@ -61,7 +61,7 @@ describe('aws-node-sdk v2auth query tests', function testSuite() {
         });
 
     it('should create a bucket', done => {
-        const params = { Bucket: bucket, Expires: 3601 };
+        const params = { Bucket: bucket, Expires: 99990 };
```

Comment:
please define 99990 as something like:
const almostOutsideTime = 99990 and let's use almostOutsideTime wherever we were using 3601 before so we will know what the numbers mean when we look back at this.